intermediateGeospatialR/src/prepWorkshopFilePrep.R

```diff
@@ -23,23 +25,23 @@ processImagery<- function(baseDir, radiance){
                      recursive = TRUE, 
                      pattern = "avg_rade9h.tif")
   }else{
-    ### proces counts data 
+    ### process counts data 
     im <- list.files(path = "F:/geoSpatialCentroid/covidNightLights/data/2019",
                      full.names = TRUE,
                      recursive = TRUE, 
                      pattern = "cf_cvg.tif")
   }
-
+}
   
 
-  
-  # pull spatial feature for the lone star star 
-  bigTex <- sf::st_read("F:/genericSpatialData/US/states/tl_2017_us_state.shp")%>%
```

Comment:
I'm going to change this so that the data is pulled directly using rNaturalEarth Package